### PR TITLE
feature/AboutUkraine-snapshot-test: created: snapshot test

### PR DESCRIPTION
**Pull request by valeriaholdovych**

### CODE REVIEW

**Review by AltysS — APPROVED** [review]

File: client/src/components/AboutUkraine/AboutUkraine.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/prop-types */
+ /* eslint-disable react/react-in-jsx-scope */
+
+ import {render} from '@testing-library/react'
+ import AboutUkraine from './AboutUkraine'
+ 
+ jest.mock('@mui/material/Box', () => ({ children }) => <section>{children}</section>);
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/components/AboutUkraine/AboutUkraine.test.js]:
```suggestion
 jest.mock('@mui/material/Box', () => ({ children }) => <div>{children}</div>);
```

File: client/src/components/AboutUkraine/AboutUkraine.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ jest.mock('@mui/material/Typography', () => ({ children }) => <h2>{children}</h2>);
+ jest.mock('../../components/AboutUkraine/AboutUkraine', () => () => <section>About Ukraine</section>);
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/components/AboutUkraine/AboutUkraine.test.js]: not needed. delete it.. why are you mocking the main component that we testing ?

Comment (valeriaholdovych) [reply client/src/components/AboutUkraine/AboutUkraine.test.js]: done

File: client/src/components/AboutUkraine/AboutUkraine.test.js (hunk 1 above)
Comment (valeriaholdovych) [reply client/src/components/AboutUkraine/AboutUkraine.test.js]: done

**Review by Ahmed-Fahmy-Dev — APPROVED** [review]

**Pull request merged**